### PR TITLE
Presse update

### LINKED ISSUE #26: Typo on main
"Lokal" Section Haus Mitbewohner\*innen > Haus-mitbewohner\*innen

### PR DESCRIPTION
**Pull request by kenodressel**
fixes #26 and adds articles + logos
also puts github on sharing icons

### CODE REVIEW

**Comment by florianschmidt1994** [comment]
Looks pretty good to me. Some minor suggestions:

*Should we add some more padding/margin here?*
image

File: src/components/AskForHelp.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default function AskForHelp () {
         Wenn dir jemand helfen möchte, kann er dich über diese Website kontaktieren und wir leiten die Kontaktanfrage automatisch an deine E-Mail.
         Alles weitere könnt ihr per E-Mail besprechen.
         <div className=" w-full rounded p-4 bg-kaki mt-4">
-          <strong>Bitte hier stellt keine ANGEBOTE rein.</strong> Wenn Du Dich benachrichten lassen willst, wenn jemand in deiner Nähe Hilfe benötigt, nutze <a
+          <strong>Bitte stellt hier keine ANGEBOTE ein.</strong> Wenn Du Dich benachrichten lassen willst, wenn jemand in deiner Nähe Hilfe benötigt, nutze <a
```
Comment (florianschmidt1994) [inline src/components/AskForHelp.js]: IMO looks a little bit aggressive in UPPERCASE. I think we could make it lowercase and it would still be dominant

**Comment by kenodressel** [comment]
You mean have a bigger distance between the logos & the articles & the download?

**Comment by florianschmidt1994** [comment]
Yes. Also I just tried to add `text-gray-700` to the PRESSE text on large screens and IMO it looks a little bit better :)

**Comment by kenodressel** [comment]
Sure add your commit :)

File: src/components/AskForHelp.js (hunk 1 above)
Comment (kenodressel) [reply src/components/AskForHelp.js]: okay lets make it "Angebote"

**Comment by florianschmidt1994** [comment]
LGTM

**Pull request merged**